### PR TITLE
add simple explanation of vscode actions

### LINKED ISSUE #627: The 'Show definition/reference/quick fix' section of the documentation could use an explain in line.
image

Just a short sentence explaining they each perform a vscode action of the same name, not all users will be aware of them. If you assign this to me I'll try to come back to it.

### PR DESCRIPTION
**Pull request by AndrewDant**
Closes #627

### CODE REVIEW

**Review by pokey — COMMENTED** [review]
I'm not sure this is much clearer. I'd lean towards a blurb on the end of each item in the list

**Comment by AndrewDant** [comment]
That's fair. I think the new commit should be a lot better.

File: docs/user/README.md [hunk 1]
```diff
@@ -448,10 +448,12 @@ See [experimental documentation](experimental/wrapper-snippets.md).
 
 ### Show definition/reference/quick fix
 
-- `"define <TARGET>"`
-- `"reference <TARGET>"`
-- `"hover <TARGET>"`
-- `"quick fix <TARGET>"`
+Each of these commands perform a vscode action of the same or a similar name on the target.
```
Comment (pokey) [inline docs/user/README.md:451]: Extremely pedantic grammatical nit (sorry 😅)
```suggestion
Each of these commands performs a vscode action of the same or a similar name on the target.
```

File: docs/user/README.md [hunk 2]
```diff
@@ -448,10 +448,12 @@ See [experimental documentation](experimental/wrapper-snippets.md).
...
+
+- `"define <TARGET>"` move the cursor and focus to the original definition of the target
```
Comment (pokey) [inline docs/user/README.md:453]:
```suggestion
- `"define <TARGET>"` navigate to the definition of the target
```

**Review by pokey — COMMENTED** [review]
Couple minor nits but otherwise looks good thanks!

**Comment by AndrewDant** [comment]
I agreed with and implemented those changes.

**Review by pokey — APPROVED** [review]

**Pull request merged**